tests/commands/test_base.py

```diff
@@ -0,0 +1,49 @@
+from unittest.mock import patch
+
+import tests
+from tests.test_base import TestBase
+
+from aws_okta_processor.commands.base import Base
+
+
+class TestCommand(Base):
+    def __init__(self, options):
```

Comment:
unittest is skipping this class due to constructor usage. Look at using unittest TestCase setUp instance method or setUpClass class method.

Reply:
Are you sure this is not being skipped because the class does not inherit from TestCase (see https://github.com/pytest-dev/pytest/blob/92767fec5122a14fbf671374c9162e947278339b/src/_pytest/unittest.py#L24)? A class type cannot be declared in a setUp method.

Reply:
Could you nest the class then pass it to TestCase class attribute?

Reply:
What is the harm in this approach?

Reply:
I'll add it to fixtures